cmd/combine.go

```diff
@@ -27,19 +27,13 @@ package cmd
 
 import (
 	"bufio"
-	"fmt"
```

Comment:
Revert all changes to this file for now so you have the original to compare against while developing the new combine package.